quickwit/quickwit-ingest/src/ingest_v2/rate_limiter.rs

```diff
@@ -63,27 +63,32 @@ pub(super) struct RateLimiter {
     refill_period: Duration,
     refill_period_micros: u64,
     refill_at: Instant,
+    throughput_num_bytes: ByteSize,
```

Comment:
it used to be more generic (not specific to bytes)

It might be good to rename it.

Reply:
Yes, good point. I'll take it out into a dedicated `RaterMeter` object.